homeassistant/components/camera/synology.py

```diff
@@ -164,10 +169,12 @@ def get_session_id(hass, username, password, login_url, valid_cert):
     }
     try:
         with async_timeout.timeout(TIMEOUT, loop=hass.loop):
-            auth_req = yield from hass.websession.get(
+            auth_req = yield from aiohttp.get(

```

Comment:
[This method has been deprecated.](http://aiohttp.readthedocs.io/en/stable/client_reference.html#aiohttp.get) You have to use a session instead.

See my example here https://github.com/home-assistant/home-assistant/issues/4257#issuecomment-258749667